app/views/katello/api/v2/content_facet/show.json.rabl

```diff
@@ -0,0 +1,12 @@
+
```

Comment:
I feel like these comments are now in the wrong place lol. I only remember commenting about the empty lines at the top of the files.